Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -137,21 +140,32 @@ public struct WorldScaleGeoTrackingSceneView: View {
         .task {
             do {
                 try await locationDataSource.start()
-                await withTaskGroup(of: Void.self) { group in
-                    group.addTask {
-                        for await location in locationDataSource.locations {
-                            lastLocationTimestamp = location.timestamp
-                            currentLocation = location
-                            await updateSceneView(for: location)
-                        }
-                    }
+                for await location in locationDataSource.locations {
+                    lastLocationTimestamp = location.timestamp
+                    currentLocation = location
+                    updateSceneView(for: location)
                 }
             } catch {}
         }
-        .toolbar {
-            ToolbarItem(placement: .bottomBar) {
-                if !calibrationViewIsHidden {
-                    calibrationView
+        .overlay(alignment: calibrationViewAlignment) {
```

Comment:
with alignment set to `.bottomLeading` this is how it looks on my phone. Needs leading padding:
![IMG_6718](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3718875/05f83566-73cc-45d6-a216-8e22b7ffc69f)
![IMG_6717](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3718875/d4b85fd1-00d6-4ec4-b0d8-b0cedfe0fb65)

Reply:
Fixed with padding around the calibration button and view.